packages/vaadin-rich-text-editor/src/vaadin-rich-text-editor.js

```diff
@@ -9,9 +9,9 @@ import { Debouncer } from '@polymer/polymer/lib/utils/debounce.js';
 import { resetMouseCanceller } from '@polymer/polymer/lib/utils/gestures.js';
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import '@vaadin/button/src/vaadin-button.js';
+import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
```

Comment:
**nit:** Sort the mixins in alphabetical order.

Reply:
Done.